modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client.cli/src/main/scala/edu/gemini/seqexec/web/client/cli/SeqexecTerminal.scala

```diff
@@ -102,17 +103,17 @@ object SeqexecTerminal extends js.JSApp {
   }
 
   object ShowHandler extends CommandHandler {
-    def width(ks: List[StepConfig]): Int = ks match {
-      case Nil => 0
-      case _   => ks.map(_.key.length).max
+    def width(ks: StepConfig): Int = ks match {
+      case m if m.isEmpty => 0
+      case _              => ks.map(_._2.length).max
     }
 
     def staticResponse(c: CliCommand) = c match {
       case SequenceConfig(_, _, _, k) =>
         val pad = width(k)
-        k.sortBy(_.key).map(s => {
-          val paddedKey = s"%-${pad}s".format(s.key)
-          s"$paddedKey -> ${s.value}"}).mkString("\n")
+        k.toList.sortBy(_._1).map(s => {
```

Comment:
Just curious, when converting a `Map` to a `List` isn't it sorted by key alphanumerically by default?